scripts/validate_docstrings.py

```diff
@@ -490,6 +514,13 @@ def validate_one(func_name):
         for param_err in param_errs:
             errs.append('\t{}'.format(param_err))
 
+    pep8_errs = [error for error in doc.validate_pep8()]
```

Comment:
I think this is easier to read:
```suggestion
    pep8_errs = list(doc.validate_pep8())
```